Add global method to get formated link

Roblinde opened the pull request.

Roblinde commented:
Failing because #17 is not merged yet

dlitvakb commented on routes/base.py:
```diff
@@ -150,6 +150,27 @@ def format_meta_title(title, locale):
         translate('defaultTitle', locale)
     )
 
+def get_custom_parameters():
```
@Roblinde In python the convention is to avoid names starting with `get_`, please change it to something like `parameterized_url` or something of that sort.

Cheers

Roblinde replied:
will do

Roblinde commented:
Added spaces! Regarding the if I disagree, it's ugly as heck, but it's (at least to me) obvious what the purpose is and it reads like normal English (more or less). Moving it to a separate function would hide that logic and make me lose context when figuring out when this URL is written or not. To add a variable for it does maybe improve the readability ever so slightly, but makes another useless allocation.

`enable_editorial_features=Enabled` is a good point. I just added it out of habit because I must set the session string to some value in csharp, but it's not necessary to have it in the query. I'll remove it.

dborsatto commented:
Cool, as soon as you push the new commit, I'll approve 🙂
